src/main/java/org/testng/internal/TestResult.java

```diff
@@ -39,6 +39,11 @@ public TestResult() {
 
   }
 
+  public TestResult(Object instance, ITestNGMethod method, Throwable throwable, ITestContext context) {

```

Comment:
Is it possible to call another constructor?

Reply:
Yes we could have. But since I didn't want to invoke `System.currentTimeMillis()` twice, I had to assign it to a variable. But since the constructor call had to be the first statement, I chose not to call the other constructor.

Reply:
ok